lib/shared/store.js

```diff
@@ -176,48 +201,32 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         })
       }
 
-      if (!action.data.type) {
-        throw errors.createError({
-          title: 'Missing state type'
-        })
-      }
+      const fields = [
```

Comment:
This should be a constant way higher up, out of the local scope; we wouldn't want an array to be allocated for this on each call